neptune_tensorflow_keras/impl/__init__.py

```diff
@@ -116,24 +119,33 @@ def _log_metrics(self, logs, category: str, trigger: str):
         if not logs:
             return
 
-        logger = self._metric_logger[category][trigger]
+        logger = self._metric_logger["training"][category][trigger]
```

Comment:
Shouldn't this be `self._metric_logger[category]` as `base_namespace` should take care of adding `training` or anything that user specified?

Reply:
@kshitij12345 it's a question for you. The spec used "training" in paths, I assumed we want this to be a fixed name?

Reply:
This discussion is resolved offline :)

Reply:
Fixed as we discussed.